cmd/_integration-tests/Makefile

```diff
@@ -7,20 +7,28 @@ TRANSPORT_TEST_MSG=\n$(OK_COLOR)Starting transport end to end test:$(END_COLOR_L
 
 CLI_TEST_MSG=\n$(OK_COLOR)Start server and cliclient generate, build, and run test:$(END_COLOR_LINE)
 
+export PATH := $(CURDIR):$(PATH)
```

Comment:
Note that `protoc-gen-truss-protocast` needs to be installed for the normal `go test ./...` tests to work, as they shell out to `protoc` and `protoc-gen-truss-protocast`. So this will only effect the integration tests in this Makefile, and if the user has never installed `protoc-gen-truss-protocast` then then those tests will fail. 

Though that binary should never change, so I am going to say it is a non issue. As this change is mostly so that the developer running the tests run the intergration tests with the current source code on disk rather than whatever truss they have in their own path.